composer.json

```diff
@@ -26,7 +26,7 @@
         "doctrine/cache": "^1.12.1 || ^2.1.1",
         "doctrine/collections": "^1.5",
         "doctrine/common": "^3.0.3",
-        "doctrine/dbal": "^2.13.1 || ^3.2",
+        "doctrine/dbal": "^2.13.1 || ^3.4@dev",
```

Comment:
This way, we test some builds twice: once as `^3.4@dev` here and once as overridden with `^3@dev` in the GitHub Actions configuration. I believe it's fine since it's a temporary state. Reworking the build matrix for the sake of purity isn't worth it.